src/utils/addMinutes.ts

```diff
@@ -0,0 +1,9 @@
+import { isNonNull } from "./null";
+
+// Adds some number of minutes to a time passed as a string in ISO format
+// Null values are treated as zeroes
+export function addMinutes(time: string, minutes: number | null | undefined) {
```

Comment:
I think there's already a function to do this in the `date-fns` library. I think that's installed.